src/api_server/swagger/firecracker.yaml

```diff
@@ -826,6 +826,8 @@ definitions:
     description:
       The CPU Template defines a set of flags to be disabled from the microvm so that
       the features exposed to the guest are the same as in the selected instance type.
+      This parameter has been deprecated and it will be removed in accordance with our
+      policy.
```

Comment:
nit: We already have 3 different ways to express "deprecated parameter" in our swagger file, could we maybe unify them all? I like the wording @bchalios  chose for the `version` field: "This parameter has been deprecated and it will be removed in future Firecracker release." We could also update the `mem_file_path` and `vsock_id` parameters to use this wording

Reply:
"in future Firecracker release" looks a bit vague to me. In the snapshot versioning case, the snapshotting feature is still in developer preview, so we don't have to follow our deletion policy of deprecated elements, but have to follow it in the static CPU template deprecation case. wdyt?

Reply:
Ahh, true, I didnt consider that. Mh, could we then make the other places that are currently also being deprecated (vsock_id and mem_file_path) to also use the wording from this PR?

Reply:
I preferred the "in a future release" approach because it is generic enough. I assume that "in accordance with our policy" is implied :), but either way I don't have strong feelings. Maybe let's try to be consistent with the wording use in other cases (if we think that it works for us).

Reply:
followed the majority vote and used the generic description.